src/pymovements/__init__.py

```diff
@@ -50,3 +50,6 @@
     'synthetic',
     'utils',
 ]
+
+from . import _version
```

Comment:
can you put it into the top and change it to `from pymovements import _version`

Reply:
done